editor/components/inserter/menu.js

```diff
@@ -211,22 +223,6 @@ export class InserterMenu extends Component {
 }
 
 export default compose(
-	withSelect( ( select ) => {
-		const {
-			getBlockInsertionPoint,
-			getInserterItems,
-			getFrecentInserterItems,
-			getSupportedBlocks,
-			getEditorSettings,
-		} = select( 'core/editor' );
-		const { allowedBlockTypes } = getEditorSettings();
-		const { rootUID } = getBlockInsertionPoint();
-		const supportedBlocks = getSupportedBlocks( rootUID, allowedBlockTypes );
-		return {
-			items: getInserterItems( supportedBlocks ),
-			frecentItems: getFrecentInserterItems( supportedBlocks ),
-		};
-	} ),
 	withDispatch( ( dispatch ) => ( {
 		fetchSharedBlocks: dispatch( 'core/editor' ).fetchSharedBlocks,
```

Comment:
Minor: It is a little strange that we are dispatching a fetch action here but we are not selecting anything in this component. To make this relationship more clear maybe the `items: getInserterItems( rootUid )` should continue to be done here and not passed as prop.

Reply:
I'll leave this for now because we're looking into replacing this action with a resolver which would address this.

Reply:
>I'll leave this for now because we're looking into replacing this action with a resolver which would address this.

Where is this tracked?